examples/examples-log4j2/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
Is this file used? Most example sub-directories don't contain gradlew (Linux shell script). Redundant?